addons/binding/pom.xml

```diff
@@ -20,6 +20,8 @@
     <module>org.openhab.binding.autelis</module>
     <module>org.openhab.binding.avmfritz</module>
     <module>org.openhab.binding.dscalarm</module>
+	<module>org.openhab.binding.feed</module>
```

Comment:
Could you please add the binding also as a Karaf feature in https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml?